### ISSUE TITLE
bogus records

### ISSUE THREAD

**Issue by eroux** [BuddhistDigitalResourceCenter/library-issues#206]
Some records are completely bogus and should be removed (I only list those with status "released", a few others can be found):
- [ ] W1AC60
- [ ] W3CN8601
- [x] W4CZ57660
- [x] W4CZ302693
- [x] W4CZ307278
- [ ] W3CN15242

**Comment by karmagongde** [comment]
These work numbers are used and using. Now almost fine except W1AC60. I used this one and know unable to release/publish. It is still on my DLMS. Anybody has any help?

**Comment by eroux** [comment]
Three of them are ok now, thanks! The remaining 3 are still quite bogus (no information at all, not even a title), maybe they should be in a provisional or accessionned state?

**Comment by karmagongde** [comment]
I already input all the catalog information on W1AC60. But I am not able to release/publish this work number. it is still on my DLMS.
W3CN8601 and W3CN15242 we are reusing.

**Comment by xristy** [comment]
@karmagongde try saving the record you've got in your Editor and checkout W1AC60 again and transfer the work you've done to the freshly checked out record. 

I checked W1AC60 out and added a test title and published it back with no problem. I didn't get the usual message suggesting that the record was already checked out by you. Maybe Travis can look at the issue if the steps I have suggested are not effective for you
